frontend/src/components/util/resetComponent.tsx

```diff
@@ -50,6 +43,14 @@ export function ResetComponent({ handler, name }: ResetComponentProps) {
     async function onConfirm() {
         if (firstEntry !== secondEntry) {
             setShowDanger(true);
+        } else if (token) {
+            const response: AxiosResponse = await handler(token, firstEntry);
+            if (response.status == StatusCodes.OK) {
+                setShowSuccess(true);
+                setTimeout(function () {
+                    router.push(transformer("/" + applicationPaths.login));
+                }, timers.redirect);
```

Comment:
Dangling promise. (Please edit this underneath here as well)

```suggestion
                setTimeout(function () {
                    router.push(transformer("/" + applicationPaths.login)).catch(console.log);
                }, timers.redirect);
```